Octokit.Reactive/Clients/ObservableOrganizationsClient.cs

```diff
@@ -35,6 +36,12 @@ public ObservableOrganizationsClient(IGitHubClient client)
         public IObservableTeamsClient Team { get; private set; }
 
         /// <summary>
+        /// A client for GitHub's Organization Hooks API.
+        /// </summary>
+        /// <remarks>See <a href="http://developer.github.com/v3/orgs/hooks/">Hooks API documentation</a> for more information.</remarks>
+        public IObservableOrganizationHooksClient Hooks { get; private set; }
```

Comment:
This should be in the singular - Hook to be consistent with the other sub-client properties (will flow from a refactor of the interface but figured I'd mention it here also)